plugins/mcreator-localization/help/fr_FR/biome/max_water_depth.md

```diff
@@ -0,0 +1,2 @@
+Ce paramètre contrôle le nombre de blocs que le tronc de l'arbre personalisé peut avoir dans l'eau.
+NOTE : Uniquement disponible pour Minecraft 1.16.4+

```

Comment:
Replace the Note with this too `NOTE: Disponible seulement pour Minecraft 1.16.x et plus`